ckcp/hack/util/util-development-patch.sh

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 The Pipeline Service Authors.
```

Comment:
```suggestion
#!/usr/bin/env bash
# Copyright 2022 The Pipeline Service Authors.
```
Add shebang directive

Also, the file doesn't have executable permissions. Please add them
```sh
100644 a2cf28725444c07b014a4504b5f2ca82507ab9c3 0       ckcp/hack/util/util-development-patch.sh
```

Reply:
done, thanks